ui/src/components/credentials/IssuedTable.tsx

```diff
@@ -165,8 +172,13 @@ export function IssuedTable() {
   ];
 
   const fetchCredentials = useCallback(
-    async (signal: AbortSignal) => {
-      setCredentials({ status: "loading" });
+    async (signal?: AbortSignal) => {
+      setCredentials((oldCredentials) =>
```

Comment:
I would like to start calling these types of params "previous...", to clarify them as distinct from "current...". Would you be able to do a Ctrl+F for "oldParams", "oldLinks" and "oldState" and replace them?

Reply:
Could we do it in a different issue and not start modifying more files that need it in this one?